### DISCUSSION TITLE
unblock activecampaign's link tracker

### DISCUSSION THREAD

**Discussion by deadcoder0904** [uBlockOrigin/uAssets#17277] [Q&A]
whenever i open indiehackers newsletter, ublock blocks it.

now i signed up for a new newsletter & it redirected with:

`https://bejakovic.lt.acemlna.com/Prod/link-tracker?redirectUrl=aHR0cHMlM0ElMkYlMkZiZWpha292aWMuY29tJTJGbXZlJTJG&sig=Dj1R5nwdwZg6EjXgnpxBaqQup43fY28bTSFsqSyDbkHA&iat=1679643239&a=||65923322||&account=bejakovic.activehosted.com&email=pO5T/YnEzx2TSHP9vFDB3GmkF4xO18Wry4/Lpewh7KPtQoiH36XCssA=:oJLDFqTxx6WcFPnShK5vGwF8e2lC+ESc&s=2946a24129b6196ab5f567be425f0a71&i=1970A2094A3A8113`

but it shows me:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/16436270/227480607-fc26bd8d-1c1c-4355-93de-3d0a9aa492c4.png">

i tried a few rules in ublock but couldn't get it to work.

google didn't help either. how do i solve this?

> note: my internet is alright but this is how the page shows

**edit: i also use cloudflare 1.1.1.1. [this discussion](https://old.reddit.com/r/CloudFlare/comments/qnralw/where_to_insert_whitelist_of_websites_that_do_not/) may or may not be relevant even tho i tried disabling clouldflare warp but it didn't work either**

**Comment by mapx-** [comment]
> whenever i open indiehackers newsletter, ublock blocks it.

Provide the steps to reproduce your issue.
(are you talking about clicking some link in the newsletter email ?!)

**Reply by gorhill** [reply]
Looks like he might be using a list which remove query parameters, and the destination site does not like that removal.

@deadcoder0904 Disclose troubleshooting information from _Support_ pane in uBO.
